### PR TITLE
CalaveraTheNine's first Goodie

### PR DESCRIPTION
**Pull request by CalaveraTheNine**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
Adds my first instant answer from tutorial.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Duckduckhacks first tutorial

**What is the data source for your Instant Answer? (Provide a link if possible)**
Duckduckhack

**Why did you choose this data source?**
Advised by the tutorial

**Are there any other alternative (better) data sources?**
No

**What are some example queries that trigger this Instant Answer?**
duckduckhack CalaveraTheNine

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
None

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Which existing Instant Answers will this one supercede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
No

**Where did you hear about DuckDuckHack? (For first time contributors)**
FireFox main page

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
duckduck

## Checklist
Please place an 'X' where appropriate.

```
[X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
[X] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
[X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[X] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [X] Google Chrome
        [X] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/CalaveraTheNine.pm [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+name "IsAwesome CalaveraTheNine";
+description "My first Goodie, it let's the world know that GitHubUsername is awesome";
+primary_example_queries "duckduckhack CalaveraTheNine";
+category "special";
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/GitHubUsername.pm";
+attribution github => ["https://github.com/GitHubUsername", "GitHubUserName"],
+;
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/CalaveraTheNine.pm]: Can you remove these `;` and terminate the `attribution` line with a `;` ?

**Comment by mintsoft** [comment]
@CalaveraTheNine Congrats on your first goodie; there's just a little bit of tidying up around line 17 to fix and then we'll get it merged!

**Comment by jagtalon** [comment]
@CalaveraTheNine looks good! :+1:

**Comment by moollaza** [comment]
@CalaveraTheNine congrats! It's live: https://duckduckgo.com/?q=duckduckhack+CalaveraTheNine&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Comment by CalaveraTheNine** [comment]
Thank you, I look forward to making more answers in the future.:thumbsup:

**Pull request merged**